docs/product-manuals/operate/deployment/authentication.md

```diff
@@ -4,59 +4,60 @@ title: Authentication
 description: "Let's take a closer look at how Operate authenticates for use."
 ---
 
-Operate provides three ways for authentication:
+Operate provides three ways to authenticate:
 
-1. Authenticate with user information stored in [Elasticsearch](#user-in-elasticsearch)
-2. Authenticate via [Camunda Cloud Single Sign-On](#camunda-cloud-single-sign-on)
-3. Authenticate via [Lightweight Directory Access Protocol (LDAP)](#ldap)
+1. User information stored in [Elasticsearch](#user-in-elasticsearch).
+2. [Camunda Cloud single sign-on](#camunda-cloud-single-sign-on).
+3. [Lightweight Directory Access Protocol (LDAP)](#ldap).
 
-By default user storage in Elasticsearch is enabled.
+By default, user storage in Elasticsearch is enabled.
 
 ## User in Elasticsearch
 
-In this mode the user authenticates with username and password, that are stored in Elasticsearch.
-**username** and **password** for one user may be set in application.yml:
+In this mode, the user authenticates with a username and password stored in Elasticsearch.
+
+The **Username** and **password** for one user may be set in `application.yml`:
 
 ```
 camunda.operate:
   username: anUser
   password: aPassword
 ```
 
-On Operate startup the user will be created if not existed before.
+On startup of Operate, the user is created if they did not exist before.
+
+By default, one user with **username**/**password** `demo`/`demo` is created.
 
-By default one user with **username**/**password** `demo`/`demo` will be created.
+Add more users directly to Elasticsearch via the index `operate-user-<version>_`. The password must be encoded with a strong bcrypt hashing function.
```

Comment:
Nice catch here! For bcrypt, what do you think about handling it like `zbctl` where we wrap it in backticks so it renders as code?

Reply:
Agreed -- leaving it as ordinary text looks a little awkward. This is a good use for a code block. Will add into glossary!